ci/d2-convert-tag.sh

```diff
@@ -29,5 +24,5 @@ then
 	set +x
 	git -c "credential.https://github.com.username=${OCEAN_D2_USER}" \
 		-c "core.askPass=./ci/askpass.sh" \
-		push "https://github.com/sociomantic-tsunami/ocean-d2.git" "$d2tag"
+		push "https://github.com/sociomantic-tsunami/ocean.git" "$d2tag"
```

Comment:
I would not use the `.auto` since having `+d2` is enough indication that it was automatically generated. When we switch completely to D2 we won't add the `+d2` and we won't have a `+d1` or anything like that.